src/pages/help-centre/components/_search-section.tsx

```diff
@@ -96,8 +97,22 @@ const SearchSection = () => {
         setSearchValue(sanitize(e.target.value))
     }
 
+    const new_all_questions: TNewQuestionsData[] = React.useMemo(() => all_questions.map((section) => {
+        return {
+            ...section,
+            category: localize(section.category),
+            questions: section.questions.map((q) => {
+                return {
+                    ...q,
```

Comment:
The variable names **section**, and **q** are not very descriptive and may be difficult to understand for someone unfamiliar with the code. Consider using more descriptive names like section_data, and question 
```suggestion
                    ...q,
```